cts/cts-exec.in

```diff
@@ -107,9 +113,10 @@ class ExecTest(Test):
         ])
 
     def _start_daemons(self):
-        if not self.tls:
+        if self._corosync:
+            self._corosync.start(kill_first=True)
```

Comment:
Not necessarily for this PR, but it looks like all callers of `self._corosync.start` pass `kill_first=True`, so it'd be worth eventually just assuming that and getting rid of the argument.

Reply:
No harm in leaving it for now, not sure what we'll do with it in the future